### ISSUE TITLE
showon doesn't work with field has multiple="true"

### ISSUE THREAD

**Issue by DeathMark** [joomla/joomla-cms#6974]
In file: libraries/joomla/form/field.php line 913
```
$showon   = explode(':', $showon, 2);
$options['class'] .= ' showon_' . implode(' showon_', explode(',', $showon[1]));
$id = $this->getName($showon[0]);
$options['rel'] = ' rel="showon_' . $id . '"';
$options['showonEnabled'] = true;
```

If field has attribute multiple="true". $id will be added "[]" at the end, so this showon doesn't work correctly.

**Comment by Fedik** [comment]
it already fixed for next Joomla 3.4.2 see #6539

**Comment by zero-24** [comment]
Thanks for the update @Fedik I' closing here as it will be fixed with the next Update.
